postcss.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  plugins: [
+    require('postcss-import'),
+    require('postcss-flexbugs-fixes'),
+    require('postcss-preset-env')({
+      autoprefixer: {
+        flexbox: 'no-2009'
+      },
+      stage: 3
+    })
+  ]
+}
```

Comment:
do we need this? we use scss right?

Reply:
> we use scss right?

Yes, but I think PostCSS is part of the toolchain by default and this file is used to configure it. So for example you can add post-processing functions on the compiled output (even if you're using SCSS)..?

Reply:
Like a minimifier I guess.